cmd/eval.go

```diff
@@ -179,43 +180,60 @@ func determineInputSpec(arguments args) (*appstudioshared.ApplicationSnapshotSpe
 	var appSnapshot appstudioshared.ApplicationSnapshotSpec
 
 	// read ApplicationSnapshot provided as a file
-	if arguments.filepath != "" {
+	filepath := arguments.filepath
+	if filepath != "" {
+		utils.Trace("Reading application snapshot from %s", filepath)
 		j, err := ioutil.ReadFile(arguments.filepath)
 		if err != nil {
+			utils.Error("Failed to read application snapshot from %s")
```

Comment:
I think we want to return error here and log/report errors centrally, this is the intent of [HACBS-633](https://issues.redhat.com/browse/HACBS-633), so the line below could be:

```go
		return nil, ecerrors.FE001.CausedBy(err).Params(filepath)
```

Where `ecerrors.FE001` would already be initialized with the message `Failed to read application snapshot from %s"`.